Fails to compile in a empty project

### Issue Report: Compilation Failure with `qubit` Dependency

**Project Setup**:
In a minimal "Hello, World" Rust project with only the `qubit` dependency, the build fails due to missing features in the `tower` crate.

**Steps to Reproduce**:
1. Create a new Rust project:
    ```sh
    cargo new hello_world
    cd hello_world
    ```
2. Add `qubit` as a dependency in `Cargo.toml`:
    ```toml
    [dependencies]
    qubit = "0.10.0"
    ```
3. Run `cargo build`.

**Observed Output**:
The build fails with the following error messages:
```plaintext
error[E0432]: unresolved import `tower::util`
   --> /path/to/cargo/registry/src/utils.rs:37:12
    |
37  | use tower::util::Oneshot;
    |            ^^^^ could not find `util` in `tower`

error[E0432]: unresolved import `tower::ServiceExt`
   --> /path/to/cargo/registry/src/utils.rs:38:5
    |
38  | use tower::ServiceExt;
    |     ^^^^^^^----------
    |     |      |
    |     |      help: a similar name exists in the module: `Service`
    |     no `ServiceExt` in the root
```

These errors indicate that `tower` is attempting to use features that are not enabled by default, such as `util`.

**Temporary Solution**:
To resolve the issue temporarily, I added `tower` as a local dependency to qubit project with the `util` feature enabled:
```sh
cargo add tower --features util
```

**Expected Solution**:
For a permanent solution, consider updating the project dependencies so that `tower` is included with the required features enabled by default.

--- a/.changes/tower-build-error.md
+++ b/.changes/tower-build-error.md
@@ -1,0 +1,5 @@
+---
+"qubit": patch
+---
+
+Enable the `util` feature on `tower` to close #82

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -27,7 +27,7 @@ jsonrpsee = { version = "0.23", features = ["server"] }
 serde = { version = "1.0.195", features = ["derive"] }
 serde_json = "1.0.117"
 tokio = { version = "1.38", features = ["rt", "rt-multi-thread"] }
-tower = "0.4"
+tower = { version = "0.4", features = ["util"] }
 ts-rs = "10.1.0"
 qubit-macros = { version = "0.6.4", path = "./crates/qubit-macros" }
 trait-variant = "0.1.2"

